rick-morty-app-client/src/components/molecules/LanguageToggle/LanguageToggle.tsx

```diff
@@ -0,0 +1,33 @@
+import React from "react";
+import { useTranslation } from "react-i18next";
+
+const LanguageToggle = () => {
+  const { i18n } = useTranslation();
+  const { t }: { t: (key: string) => string } = useTranslation();
+
+  const changeLanguage = (language: string) => {
+    i18n.changeLanguage(language);
+    localStorage.setItem("language", language);
+  };
+
+  return (
+    <div>
+      <p>{t("header.welcomeMessage")}</p>
+      <button
+        onClick={() => changeLanguage("en")}
+        aria-label="Switch to English"
```

Comment:
Skoro dodajesz aria- to one tez powinny byc w tym jezyku

Reply:
poprawione

Reply:
chodzilo mi ze to abrdziej powinno byc w srodku 
t(id.translacji Dla ARIA)
i zeby zmienic jezzyk to te buttony moze w petli z dostpenych jezykow a nei tak zahardcodowane